admin/release-packaging/README.rst

```diff
@@ -0,0 +1,3 @@
+Packages for Flocker that aren't available in each supported Operating System.
```

Comment:
Please could you suggest an ideal comment - I'm not 100% clear on what this does.

I think "Packages which point to repositories where Flocker packages are stored".